apis/bigtable/v1alpha1/logicalview_types.go

```diff
@@ -0,0 +1,101 @@
+	   object's current state. */
+	Conditions []v1alpha1.Condition `json:"conditions,omitempty"`
+
+	// ObservedGeneration is the generation of the resource that was most recently observed by the Config Connector controller. If this is equal to metadata.generation, then that means that the current reported status reflects the most recent desired state of the resource.
+	ObservedGeneration *int64 `json:"observedGeneration,omitempty"`
+
+	// A unique specifier for the BigtableLogicalView resource in GCP.
+	ExternalRef *string `json:"externalRef,omitempty"`
+
+	// The unique name of the BigtableLogicalView. Values are of the form
+	//  `projects/{project}/instances/{instance}/logicalViews/{logicalViewID}`.
+	// +kcc:proto:field=google.bigtable.admin.v2.LogicalView.name
+	Name *string `json:"name,omitempty"`
+
+	// ObservedState is the state of the resource as most recently observed in GCP.
+	ObservedState *BigtableLogicalViewObservedState `json:"observedState,omitempty"`
+}
+
+// BigtableLogicalViewObservedState is the state of the BigtableLogicalView resource as most recently observed in GCP.
+// +kcc:proto=google.bigtable.admin.v2.LogicalView
+type BigtableLogicalViewObservedState struct{}
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:categories=gcp,shortName=gcpbigtablelogicalview;gcpbigtablelogicalviews
+// +kubebuilder:subresource:status
+// +kubebuilder:metadata:labels="cnrm.cloud.google.com/managed-by-kcc=true";"cnrm.cloud.google.com/system=true"
+// +kubebuilder:printcolumn:name="Age",JSONPath=".metadata.creationTimestamp",type="date"
+// +kubebuilder:printcolumn:name="Ready",JSONPath=".status.conditions[?(@.type=='Ready')].status",type="string",description="When 'True', the most recent reconcile of the resource succeeded"
+// +kubebuilder:printcolumn:name="Status",JSONPath=".status.conditions[?(@.type=='Ready')].reason",type="string",description="The reason for the value in 'Ready'"
+// +kubebuilder:printcolumn:name="Status Age",JSONPath=".status.conditions[?(@.type=='Ready')].lastTransitionTime",type="date",description="The last transition time for the value in 'Status'"
+
+// BigtableLogicalView is the Schema for the BigtableLogicalView API
+// +k8s:openapi-gen=true
+type BigtableLogicalView struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// +required
+	Spec   BigtableLogicalViewSpec   `json:"spec,omitempty"`
+	Status BigtableLogicalViewStatus `json:"status,omitempty"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// BigtableLogicalViewList contains a list of BigtableLogicalView
+type BigtableLogicalViewList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []BigtableLogicalView `json:"items"`
+}
+
+func init() {
+	SchemeBuilder.Register(&BigtableLogicalView{}, &BigtableLogicalViewList{})
```

Comment:
Rerun the `dev/tasks/generate-crds` script should fix the "DeepCopyObject" errors.